Editor/TooltipsEditor.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
```

Comment:
It is a centralized place to edit serialized tooltip strings and show the user where tooltips exist within the codebase. It's a potential first step toward being able to localize Tooltips, and generally helps us keep track of them.